### PR TITLE
Toggle button to TS and make width adaptive

### PR DESCRIPTION
**Pull request by qubis741**
### Changed

- `Toggle`: Changed width from full to adaptive

### CODE REVIEW

**Review by JorenSaeyTL — DISMISSED** [review]
MCing ⌛  ...

**Comment by JorenSaeyTL** [comment]
I'll read that branch name before I'll go to bed 😅

**Comment by JorenSaeyTL** [comment]
MC ✅ 
- Toogle still works
- When I make the label `width: 100%` I can still click on the white space of the label to change the toggle, so my question:
Is it ok that the label is clickable to change toggle? Perhaps something to check off with Stephen and someone from QA?

File: src/components/toggle/Toggle.tsx [hunk 1]
```diff
@@ -1,105 +1,93 @@
...
+  disabled = false,
+  maxLines,
+  className,
+  label,
+  onChange,
+  size = 'medium',
+  ...others
+}: ToggleProps) => {
+  const ref = useRef<HTMLInputElement>(null);
+  const restProps = omit(others, ['onChange']);
```
Comment (lorgan3) [inline src/components/toggle/Toggle.tsx:36]: I don't think this `omit` does anything, `onChange` is already destructured on line 31

File: src/components/toggle/theme.css [hunk 2]
```diff
@@ -22,8 +22,10 @@
 
 .toggle {
   align-items: flex-start;
-  display: flex;
+  display: inline-flex;
   user-select: none;
+  /* Fix for whitespace bellow, caused be inline-flex */
```
Comment (lorgan3) [inline src/components/toggle/theme.css:27]:
```suggestion
  /* Fix for whitespace below, caused be inline-flex */
```

**Comment by qubis741** [comment]
> MC ✅
> 
> * Toogle still works
> * When I make the label `width: 100%` I can still click on the white space of the label to change the toggle, so my question:
>   Is it ok that the label is clickable to change toggle? Perhaps something to check off with Stephen and someone from QA?
>   ⚠️

Yes that is desired outcome

**Review by JorenSaeyTL — APPROVED** [review]

**Review by lorgan3 — APPROVED** [review]

**Pull request merged**